salttesting/case.py

```diff
@@ -330,22 +333,26 @@ def detach_from_parent_group():
                     pass
             # pylint: enable=maybe-no-member
 
-        data = process.communicate()
-        process.stdout.close()
+        # TODO Remove this?
+        process.communicate()
```

Comment:
It should allow for stdin handling which is the only use case I can see, but since we don't do any of that here I think it's basically a no-op.